doc/release/1.15.0-notes.rst

```diff
@@ -2,109 +2,149 @@
 NumPy 1.15.0 Release Notes
 ==========================
 
+NumPy 1.15.0 is a release with an unusual number of cleanups, many deprecations
+of old functions, and improvements to many existing functions. Please read the
+detailed descriptions below to see if you are affected.
+
+For testing, we have switched to pytest as a replacement for the no longer
+maintained nose framework. The old nose based interface remains for downstream
+projects who may still be using it.
+
+The Python versions supported by this release are 2.7, 3.4-3.6.  The upcoming
+3.7 release should also work, but you will need to compile from source using
+Cython 0.28.2 or later. The wheels will be linked with OpenBLAS 3.0, which
```

Comment:
Could it be that you meant OpenBLAS 0.3 here, which was released in May this year?